src/test/java/com/webforj/samples/views/alert/ClosableAlertViewIT.java

```diff
@@ -0,0 +1,33 @@
+package com.webforj.samples.views.alert;
+
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.microsoft.playwright.Locator;
+import com.microsoft.playwright.Page;
+import com.microsoft.playwright.options.AriaRole;
+import com.webforj.samples.views.BaseTest;
+
+public class ClosableAlertViewIT extends BaseTest {
+
+    @BeforeEach
+    public void setupClosableAlert() {
+        navigateToRoute("closablealert");
+        // Kill animations for more reliable testing
+        page.addStyleTag(new Page.AddStyleTagOptions()
```

Comment:
@jturfanbasis Please don't copy and paste code snippets like this. If it's really needed everywhere, move it to **BaseTest** instead.
Also, the proper way to disable animations is by using `setReducedMotion(ReducedMotion.REDUCE)`. That should handle it and turn off the CSS animations in DWC. Have you tried that?

Reply:
https://dwc.style/docs/#/dwc/guide/themes?id=animations